File: KnowledgeCenterServer/KnowledgeCenter.DataConnector/Entities/Match/MatchingScorePerSkill.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace KnowledgeCenter.DataConnector.Entities.Match
{
    [Table("Match.MatchingScorePerSkill")]
    public class MatchingScorePerSkill
    {
        public int Id { get; set; }
        public int MatchingId { get; set; }
        public Matching Matching { get; set; }
        public int SkillLevelId { get; set; }
        public SkillLevel SkillLevel { get; set; }
        public int CustomerOfferSkillId { get; set; }
        public CustomerOfferSkill CustomerOfferSkill { get; set; }
        public double Score { get; set; }
    }
}

File: KnowledgeCenterClient/src/app/match/models/MatchingScorePerSkill.ts
import { SkillLevel } from './SkillLevel';
import { CustomerOfferSkill } from './CustomerOfferSkill';

export interface MatchingScorePerSkill {
  id: number;
  skillLevelId: number;
  skillLevel: SkillLevel;
  customerOfferSkillId: number;
  customerOfferSkill: CustomerOfferSkill;
  score: number;
}
